packages/react/src/components/Alert/index.tsx

```diff
@@ -3,7 +3,7 @@ import classnames from 'classnames';
 import Icon from '../Icon';
 import { Dialog, DialogContent, DialogFooter, DialogProps } from '../Dialog';
 
-interface AlertProps extends Omit<DialogProps, 'forceAction'> {
+interface AlertProps extends Omit<Omit<DialogProps, 'forceAction'>, 'onClose'> {
```

Comment:
Not a big deal, but I believe you can also do `Omit<DialogProps, 'forceAction' | 'onClose'>`

Reply:
Oh neat! I'll give it a try.